explorer/utils/asyncpool.ts

```diff
@@ -0,0 +1,37 @@
+/*Custom asyncpool
+ * To limit the number of asynchonous calls being done, this is helpful to limit 
+ * Connection requests and avoid throttling.
+ */
+export class asyncPool {
```

Comment:
It looks the class name convention in the project is PascalCase. Can you change it to AsyncPool?